source/Meadow-Filter-Parse.js

```diff
@@ -160,6 +160,47 @@ const addFilterStanzaToQuery = (pFilterStanza, pQuery) =>
 			}
 			break;
 
+    case 'FBJV':// Filter by JSON Value (left-side AND)
+      {
+      const fields = pFilterStanza.Field.split(',', 2);
+      const jsonField =fields[0];
+      const subFields = fields[1];
+      if (jsonField !== undefined && jsonField.trim() !== "" && subFields !== undefined && subFields.trim() !== "")
+        {
+      pQuery
+        .addFilter('', '1', `LENGTH(${jsonField}) >`, 'AND', 'jsonFieldLen')
+        .addFilter(
+          `JSON_EXTRACT(${jsonField}, '$.${subFields}')`,
+          pFilterStanza.Value,
+          getFilterComparisonOperator(pFilterStanza.Operator),
+          'AND',
+          'jsonFieldParameter'
+        );
+      }
+      else{
+          throw new Error("Invalid Jsonfield or path");
+      }
+      break;
+      }
+
+    case 'FSJF':// Filter Sort Field
+      {
+      const tmpSortDirection = pFilterStanza.Operator === 'DESC' ? 'Descending' : 'Ascending';
+      const fields = pFilterStanza.Field.split(',', 2);
+      const jsonField =fields[0];
+      const subFields = fields[1];
+      if (jsonField !== undefined && jsonField.trim() !== "" && subFields !== undefined && subFields.trim() !== "")
+        {
```

Comment:
It's weird that we are trimming here but not for the actual value used. If we need that, it feels better to do all of this during the extraction. ex.

```javascript
const jsonField = (fields[0] || '').trim();
const subFields = (fields[1] || '').trim();
```

Also (nit) generally, I would prefer to write this as a guard clause, where we bail out if we can't continue (reduces indentation).